layers/meta-resin-generic/conf/samples/bblayers.conf.sample

```diff
@@ -14,7 +14,7 @@ BBLAYERS ?= " \
     ${TOPDIR}/../layers/meta-openembedded/meta-python \
     ${TOPDIR}/../layers/meta-resin/meta-resin-common \
     ${TOPDIR}/../layers/meta-resin/meta-resin-pyro \
-    ${TOPDIR}/../layers/meta-resin-intel \
+    ${TOPDIR}/../layers/meta-resin-generic \
```

Comment:
I think this should have another name, like "meta-resin-genericx86-64"

Reply:
What if we would also support the 32 bit version? maybe a meta-resin-genericx86 would be better, what do you think?

Reply:
Agreed

Reply:
+1